using-timescaledb/continuous-aggregates.md

```diff
@@ -0,0 +1,330 @@
+When designing continuous aggregates, we wanted to avoid write amplification as
+that could drastically affect insert rate for write-heavy time-series workloads.
+We also wanted to deal gracefully with out-of-order data. There are several
+tunable parameters that allow us to control how quickly data is materialized
+after insert and how far behind the aggregates will remain (so as not to cause
+the same materialization to be modifiied multiple times and cause write amplification). 
+
+We usually stay at least one `bucket_width` (the first argument to the
+`time_bucket` call in the view definition) behind the maximum inserted time
+value. The `refresh_lag` parameter determines how much further behind the
+highest inserted time value the background worker will attempt to materialize
+(we will be behind by `refresh_lag` + `bucket_width`).  So in our example, if
+the `refresh_lag` is 1 hour and the `bucket_width` is 1 hour, the
+materialization will generally be 2 hours behind the maximum inserted value.
+Tuning the `refresh_lag` parameter lower, or even negative, will mean that the
+aggregates will follow inserts more closely, but can cause some write
+amplification which may degrade insert performance. 
+
+The `refresh_interval` controls how frequently materialization jobs will be
+launched. Setting a shorter interval will mean materializations happen more
+"eagerly" but can consume background worker resources while it is running.
+
+The `timescaledb.max_interval_per_job` parameter can also be useful, especially
+when a continuous aggregate is created on a hypertable that already contains
+data and the materialization job needs to catch up. It determines the maximum
+amount of data processed in a single job. If there is still more work to do to
+catch up to where we need to be based on the `refresh_lag` setting, another job
+will be spawned immediately following completion. This means that the part of the
+data processed in previous jobs will be available even before the
+materialization is fully caught up.
+
+>:TIP: Most times the continuous aggregate view will be updated by the background job;
+  however, if you would like to run it yourself, you may use the [`REFRESH MATERIALIZED VIEW` 
+  command][api-refresh-continuous-aggs].
+
+**Using `timescaledb.information` Views** 
+The various options used to create the continuous aggregate view, as well as its
+definition, can be found in the [`timescaledb_information.continuous_aggregates`
+view][api-continuous-aggregates-info], and information about the state and progress
+of the materialization background worker jobs can be found in the
+[`timescaledb_information.continuous_aggregate_stats`
+view][api-continuous-aggregate-stats]. These views can be quite useful for
+administering continuous aggregates and tuning other options noted below. 
+
+---
+
+### Altering and Dropping a Continuous Aggregate View [](alter-drop)
+In addition to these important options, there are several other options that may
```

Comment:
How about rephrasing it as: "In addition to the options discussed earlier", there are... . (No immediate reference to the options?)